tests/test_cursor.py

```diff
@@ -42,11 +42,21 @@ def test_format_operation_without_parameters(parameters):
     operation = "SELECT * FROM TEST WHERE fuu = 'bar'"
     assert format_operation(operation, parameters) == operation
 
-def test_format_operation_with_parameters():
+def test_format_operation_with_format_parameters():
```

Comment:
I know there are currently no tests which tests the format operation with multiple parameters but with the additional parameter styles we should add this tests.